### DISCUSSION TITLE
The sound will stutter while i am useing FxSound

### DISCUSSION THREAD

**Discussion by LoftyShadow** [LizardByte/community#373] [Sunshine]
The sound will stutter while i am useing Sunshine, and i find out it's because i am using [FxSound](https://www.fxsound.com/) 
I have searched the existing issues and discussions, and i haven't found anybody describe it
so i add this to tell people who are in the same situation as me how to solve it
here are someway to solve it
- exit FxSound
![image](https://github.com/LizardByte/Sunshine/assets/43536131/624e435d-a58f-4053-adff-be393bda4332)
- turn off FxSound, you can also use the keyboard shortcuts
![image](https://github.com/LizardByte/Sunshine/assets/43536131/133dfecb-12ad-4ba5-a87c-9ad96035825e)
- uninstall FxSound
and i think it's a FxSound bug, so i didn't create a new issue
if you have a better way to fix this, please tell me, thx
English is not my native language; please excuse typing errors.

**Comment by iamppzx** [comment]
I uninstalled fx sound but it not solve the problem. sound keep  stuttering pls help

[a comment by an automated account is not shown]
